### PR TITLE
Modifying copy process for test addon

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

**Comment by BirmacherAkos** [comment]
A test would be nice in the bitrise.yml to check if the expected export was successfully or not.

File: main.go [hunk 1]
```diff
@@ -613,18 +612,16 @@ func main() {
 	}
 
 	// exporting xcresult only if test result dir is present
-	if testResultPath := os.Getenv(bitriseConfigs.BitriseTestResultDirEnvKey); len(testResultPath) > 0 {
```
Comment (godrei) [inline main.go]: pls do a dep update, the env key is renamed and may the value also changed: `BitrisePerStepTestResultDirEnvKey`

File: result_addon.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type addonCopy struct {
+	sourceTestOutputDir   string
+	targetAddonPath       string
+	targetAddonBundleName string
+}
+
+func copyToResultAddonDir(info addonCopy) error {
```
Comment (godrei) [inline result_addon.go]: could be 2 functions, just like here: https://github.com/bitrise-io/bitrise/pull/647/files/1486ee52ac69d237d775bc366f1677555cee67a6#r252625893

may the first copy function could return the addonPerStepOutputDir and the second could use the dir to save the testBundle

File: Gopkg.lock [hunk 3]
```diff
@@ -3,31 +3,31 @@
...
 [[projects]]
-  digest = "1:73efbb654761e7e4087e4d9b912cfca51aed11329ab31888a377b46e1a3e065e"
+  digest = "1:c26c6f48d8add3ddfe9b3abf7b5d8fe2eb8f3d59f3e31e59f4488a23663e8b6b"
   name = "github.com/bitrise-io/bitrise"
   packages = ["configs"]
   pruneopts = "UT"
-  revision = "5d52fad1a1fbe34417296f1fbf6d3672c5d5efc9"
-  version = "1.27.1"
+  revision = "a7523d99944aaf56c644dec41f374f746db9a453"
+  version = "1.29.0"
```
Comment (godrei) [inline Gopkg.lock]: for some reason "github.com/bitrise-io/bitrise" is not part of the `Gopkg.toml`, could you please add it and lock for the `master` branch

**Pull request merged**